ironfish/src/wallet/database/accountsdb.ts

```diff
@@ -65,6 +69,16 @@ export class AccountsDB {
     value: Buffer
   }>
 
+  sequenceToNoteHash: IDatabaseStore<{
+    key: [Account['prefix'], [number, TransactionHash]]
```

Comment:
`TransactionHash` should be some kind of note hash type or Buffer

Reply:
fixed